packages/ui/src/pages/SourceCode/SourceCodePage.tsx

```diff
@@ -19,11 +19,5 @@ export const SourceCodePage: FunctionComponent = () => {
     [entitiesContext],
   );
 
-  useEffect(() => {
-    return entitiesContext?.eventNotifier.subscribe('code:update', (code) => {
-      setLocalSourceCode(code);
-    });
-  }, [entitiesContext?.eventNotifier, setLocalSourceCode]);
-
```

Comment:
Moved from here to the `Shell` component, as this component no longer exists when there's a route change